Resources/Prototypes/DeltaV/Catalog/Fills/Crates/arena.yml

```diff
@@ -0,0 +1,217 @@
+            range: 1, 4
+        - id: MobCarpMagic
+          amount: !type:RangeNumberSelector
+            range: 1, 4
+        - id: MobCarpHolo
+          amount: !type:RangeNumberSelector
+            range: 1, 4
+        - id: MobCarpRainbow
+          amount: !type:RangeNumberSelector
+            range: 1, 4
+        - id: MobShark
+          weight: 0.2
+          amount: !type:RangeNumberSelector
+            range: 1, 2
+      - !type:GroupSelector
+        children:
+        - id: MobPurpleSnake
+          amount: !type:RangeNumberSelector
+            range: 1, 3
+        - id: MobCobraSpace
+          amount: !type:RangeNumberSelector
+            range: 1, 3
+        - id: MobSmallPurpleSnake
+          amount: !type:RangeNumberSelector
+            range: 1, 6
+          weight: 0.4
+      - !type:GroupSelector
+        children:
+        - id: MobArgocyteSkitter
+          amount: !type:RangeNumberSelector
+            range: 1, 4
+        - id: MobArgocyteSwiper
+          amount: !type:RangeNumberSelector
+            range: 1, 3
+        - id: MobArgocyteMolder
+          amount: !type:RangeNumberSelector
+            range: 1, 3
+        - id: MobArgocytePouncer
+          amount: !type:RangeNumberSelector
+            range: 1, 3
+        - id: MobArgocyteGlider
+          amount: !type:RangeNumberSelector
+            range: 1, 2
+        - id: MobArgocyteHarvester
+          amount: !type:RangeNumberSelector
+            range: 1, 2
+        #Nanotrasen has not yet developed the technology to hold these creatures
+        #- id: MobArgocyteCrawler
+        #  amount: !type:RangeNumberSelector
+        #    range: 1, 2
+      - !type:GroupSelector
+        children:
+        - id: MobWatcherLavaland
+          amount: !type:RangeNumberSelector
+            range: 1, 2
+        - id: MobWatcherIcewing
+          amount: !type:RangeNumberSelector
+            range: 1, 2
+        - id: MobWatcherMagmawing
+          amount: !type:RangeNumberSelector
+            range: 1, 2
+        - id: MobWatcherPride
+          amount: !type:RangeNumberSelector
+            range: 1, 2
+          weight: 0.1
+      - !type:GroupSelector
+        children:
+        - id: MobAdultSlimesBlueAngry
+          amount: !type:RangeNumberSelector
+            range: 1, 3
+        - id: MobAdultSlimesGreenAngry
+          amount: !type:RangeNumberSelector
+            range: 1, 3
+        - id: MobAdultSlimesYellowAngry
+          amount: !type:RangeNumberSelector
+            range: 1, 3
+      - id: MobTomatoKiller
+        amount: !type:RangeNumberSelector
+          range: 2, 5
+    #Uncommon
+    - !type:GroupSelector
+      weight: 0.5
+      children:
+      - id: MobLaserRaptor
+        weight: 10
+      - id: MobLaserRaptor
+        amount: !type:ConstantNumberSelector
+          value: 2
+      - id: MobFleshClamp
+        amount: !type:ConstantNumberSelector
+          value: 3
+
+- type: entityTable
+  id: LargeArenaMobTable
```

Comment:
12k to get a single ore crab is ridiculous
are you sure you only want this to spawn 1 mob (GroupSelector rolls 1 child, AllSelector adds all children)

Reply:
Yes, the crab is currently set up how I want it. 

I originally priced it at 6k but arbitrage told me I had to go to 12.5k. I could either figure out which mobs are jacking up the price so much and moving them to their own crate, or move crabs to the small mob crate. they are kinda medium I think.